sklearn/model_selection/tests/test_search.py

```diff
@@ -490,16 +490,18 @@ def test_grid_search_sparse():
     assert_equal(C, C2)
 
 
+@pytest.mark.filterwarnings('ignore: The default of the `iid`')  # 0.22
```

Comment:
Is this `iid` warning being raise in this test?

Reply:
@thomasjpfan No, this warning is not raised. Still, this warning is not raised either by the following test (`test_grid_search_precomputed_kernel`) and it bears this this decorator. By looking at the other tests in this file (`test_search.py`), I thought that tests with `GridSearchCV` are expected to have these two pytest decorators.